pom.xml

```diff
@@ -17,19 +17,34 @@
     <version>5.2.0</version>
 
     <name>Example</name>
-    <description>An example maven project for connection to OMERO</description>
+    <description>An example maven project for connection to OMERO using the Java gateway.</description>
 
     <dependencies>
         <dependency>
           <groupId>omero</groupId>
           <artifactId>blitz</artifactId>
+          <version>5.2.0-ice35-b12</version>
         </dependency>
         <dependency>
           <groupId>ome</groupId>
           <artifactId>bioformats_package</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Shouldn't these be pulled in by some "gateway" dependency if not "blitz" itself?